.circleci/main.yml

```diff
@@ -542,6 +550,45 @@ jobs:
       - store_artifacts:
           path: /tmp/debug-dump
           destination: debug-dump
+  create-nightly-tag:
+    executor: docker-go
+    steps:
+      - checkout
+      - add_ssh_keys:
+          fingerprints:
+            - "4c:1c:ff:56:c1:d0:07:03:23:08:08:2e:e6:16:30:1d"
+      - run:
+          name: setup pachydermbuildbot git
+          command: |
+            git config --global user.email "buildbot@pachyderm.io"
+            git config --global user.name "pachydermbuildbot"
+      - run:
+          name: create nightly tag if new commits
+          command: |
+            changes=$( env -i git log --pretty=oneline --since='yesterday' | wc -l )
+            if [ $changes -gt 0 ]; then
+              TIMESTAMP=$(date +'%Y%m%d%T' | sed 's/://g')
```

Comment:
Do we need the timestamp (%T) in here?  If we release once a day, %Y%m%d should be sufficient?  Also, what time zone is the date in?  Does it depend on the worker machine at all?